sklearn/linear_model/_logistic.py

```diff
@@ -1108,10 +1110,14 @@ class LogisticRegression(LinearClassifierMixin,
         - For multiclass problems, only 'newton-cg', 'sag', 'saga' and 'lbfgs'
           handle multinomial loss; 'liblinear' is limited to one-versus-rest
           schemes.
-        - 'newton-cg', 'lbfgs', 'sag' and 'saga' handle L2 or no penalty
-        - 'liblinear' and 'saga' also handle L1 penalty
-        - 'saga' also supports 'elasticnet' penalty
-        - 'liblinear' does not support setting ``penalty='none'``
+
```

Comment:
I propose to use to go further and add a table (we will have to check the rendering thought):

```
        Algorithm to use in the optimization problem. The solver choice will
        also be dependent of the `penalty` and `multi_class` parameters. The
        table below summarizes the option supported for each solver. A `X`
        indicates that this option is supported.

        +-------------+------------+----+----+-----------+-------------+
        |   Solver    | No penalty | L1 | L2 | ElastiNet | Multinomial |
        +=============+============+====+====+===========+=============+
        | 'newton-cg' |      X     |    | X  |           |      X      |
        +-------------+------------+----+----+-----------+-------------+
        |   'lbfgs'   |      X     |    | X  |           |      X      |
        +-------------+------------+----+----+-----------+-------------+
        | 'liblinear' |            | X  | X  |           |             |
        +-------------+------------+----+----+-----------+-------------+
        |    'sag'    |      X     |    | X  |           |      X      |
        +-------------+------------+----+----+-----------+-------------+
        |    'saga'   |      X     | X  | X  |     X     |      X      |
        +-------------+------------+----+----+-----------+-------------+

        .. note::
           For small datasets, `'liblinear'` is a good choice, whereas `'sag'`
           and `'saga'` are faster for large ones.

        .. note::
           `'sag'` and `'saga'` fast convergence is only guaranteed on
           features with approximately the same scale. You can
           preprocess the data with a scaler from :mod:`sklearn.preprocessing`.
```

Reply:
I wouldn't go that way - We would not be 100% sure if it is displayed correctly due to lots of variables (e.g editors and IDEs could use different fonts where used signs have diffrent sizes). 

@thomasjpfan - Since it provides more specific knowledge - How about using the link as 'More Info' or 'Additional Souce'?

Reply:
> I wouldn't go that way - We would not be 100% sure if it is displayed correctly due to lots of variables (e.g editors and IDEs could use different fonts where used signs have diffrent sizes).

In the worst-case scenario, it will be displayed as pure text and it should be fine.

> We already have a table in the User Guide. What do you think about creating an anchor on that table and linking it here?

Until by doing `help(LogisticRegression)` we get the information in text format, I am fine with it.